### PR TITLE
Menu

### LINKED ISSUE #137: Create Menu

### PR DESCRIPTION
**Pull request by Dezerack**
Closes #137

### CODE REVIEW

**Comment by BrandonPacewic** [comment]
Seems to work for `Example Room 1` but not `Example Room 2`?

**Comment by Galaxy25** [comment]
@BrandonPacewic Made a lot of fixes. It should be good now

**Review by Galaxy25 — DISMISSED** [review]

File: Assets/Scripts/UI/AmmoCounter.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+// SPDX-License SPDX-License-Identifier: Apache-2.0
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class AmmoCounter : MonoBehaviour
+{
+    public TextMeshProUGUI bulletText;
```
Comment (BrandonPacewic) [inline Assets/Scripts/UI/AmmoCounter.cs:11]: I believe that this variable needs to be renamed.

**Review by BrandonPacewic — CHANGES_REQUESTED** [review]

File: Assets/Scripts/UI/AmmoCounter.cs (hunk 1 above)
Comment (Galaxy25) [reply Assets/Scripts/UI/AmmoCounter.cs:11]: To what?

Comment (BrandonPacewic) [reply Assets/Scripts/UI/AmmoCounter.cs:11]: Just needs to be capitalized right?

**Comment by Galaxy25** [comment]
@BrandonPacewic Renamed it

**Comment by BrandonPacewic** [comment]
Looks good to me! Going to wait to merge till we finish #141 so we can avoid adding the 55,000+ lines for text mesh pro.

**Comment by BrandonPacewic** [comment]
See #151.

**Review by Galaxy25 — APPROVED** [review]

**Review by BrandonPacewic — APPROVED** [review]
I'm getting ready to merge and updating the change log - please @me if any other changes are pushed.

**Pull request merged**